ocs_ci/helpers/helpers.py

```diff
@@ -1392,6 +1392,55 @@ def measure_pvc_creation_time_bulk(interface, pvc_name_list, wait_time=60):
     return pvc_dict
 
 
+def measure_multiple_pvc_deletion_time(interface, pvc_name, status):
+    """
+    Get the starting/ending deletion time of a PVC based on provisioner logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_name (str / list): Name of the PVC(s) for creation time
+                               the list will be list of pvc objects
+        status (str): the status that we want to get - Start / End
```

Comment:
Purpose of the function is to return start time or end time of PVC ? if so function name is misleading please change it to something like get_multiple_pvc_start_or_end_time()